test_suite/tests.source

```diff
@@ -18,9 +18,36 @@
 EXECUTED=0
 FAILED=0
 
-# Display successful as well as failed tests (useful if they are slow).
-VERBOSE=false
+# Display additional information such as successful tests (useful if they are slow).
+VERBOSE="${VERBOSE:-false}"
 
+separator() {
+    echo -e "\n-----------------------------\n"
+}
+
+get_temp_dir() {
+    if [ -z "$TMP_DIR" ]; then
+        # Simplest and most portable way I could find.
+        TMP_DIR="`python2 -c "import tempfile; print(tempfile.mkdtemp(prefix='jsonnet_'))"`"
```

Comment:
Yes. There are portability problems. OS X and GNU versions of `mktemp` need different arguments. Invoking Python seems much cleaner to me than trying to autodetect the version or trying the other version if it fails (the latter is the top answer here https://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x).